### ISSUE TITLE
[charts] auto chart-bump improvements

### ISSUE THREAD

**Issue by nicholasSUSE** [rancher/ecm-distro-tools#517]
The auto-chart-bump is executed manually using the [charts-build-scripts](https://github.com/rancher/charts-build-scripts). 

This will be the list of improvements to implement before we can make this feature mandatory for every chart owner and make an announcement company-wide. 

---

### Improvements

1. Implement auto-trigger at chart-bump.

When a new PR is merged on the chart upstream repository, this must trigger a new auto chart-bump at the charts repository. 

This must be done with @pjbgf approval, the strategy that will be implemented must not generate security risks. 

This must be documented in the Wiki before the announcement.

After the company-wide statement the Chart owners will be responsible for implementing their part of the integration in their upstream repositories respectively. 

2. When a new chart-bump occurs, the last `release.yaml` entry for that chart must be removed before a new one is added. 
3. When a new chart-bump occurs the auto-chart-bump code must return the version to be bumped so we can display it on CI and create a branch with the version and also add it to the PR Body. 
4. The feature currently is hard-coded only for `dev-v2.10` branch, expand the functionality for the newer lines that might come (e.g., `dev-v2.11`; `dev-v2.12`....)

**Comment by nicholasSUSE** [comment]
Step 1 is done as in: https://github.com/rancher/charts/pull/4999

**Comment by nicholasSUSE** [comment]
Adding another step. 

Fix once and for all Airgap Icon problems like: https://github.com/rancher/rancher/issues/45262

There will be no option for adding local Icons or not. 
Everything on Auto Bump will automatically download and referto  the local icon paths.
